src/components/Button/Button.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import styles from "./Button.module.css";
+
+function Button({ children, isButtonActive, ...props }) {
+  return (
+    <button
+      className={`${styles.Button} ${isButtonActive ? styles.active : ""}`}
+      {...props}
```

Comment:
Bu bir `Button` bileşeni, bu yüzden `active` olduğunu bildirmek için `isButtonActive` demeye ihtiyacımız yok.

`active` ya da `isActive` tercih edebilirsin.

`CSS Modules` deki `composes` özelliğini incelersen bunu şu şekilde yazabilirsin.

```jsx
    <button
      className={active ? styles.ButtonActive : styles.Button}
```